src/content/docs/usage/Animating/procedural.mdx

```diff
@@ -9,125 +9,156 @@ import sophGraphImg from 'assets/bopGraph.png';
+        isContextful: true,
+        default: "0.0"
+      }
+    ],
+    description: "Applies an easing function to a value, transforming it based on the selected easing type. This is useful for creating smooth transitions in animations."
+  }}/></a>
 
 <br/>
 
-<a style={{textDecoration: "none"}} href="/docs/usage/node-graph/nodes/animation/easing"><Node data={{
-  name: "Easing ",
-  category: "Animation",
-  icon: "icon-spline-chart",
-  isPair: false,
-  hasPreview: false,
-  inputs: [
-    {
-      name: "Value",
-      type: "Double",
-      description: "The input value to be eased. This value should be between 0 and 1.",
-      isContextful: true,
-      default: "0.0"
-    },
-    {
-      name: "Easing Type",
-      type: "EasingType (Enum)",
-      hideSocket: true,
-      typeLink: "/docs/usage/node-graph/nodes/animation/easing#easing-types",
-      description: "The type of easing function to apply to the input value. This determines how the value transitions over time.",
-      isContextful: false,
-      default: "Linear"
-    }
-  ],
-  outputs: [
-    {
-      name: "Output",
-      type: "Double",
-      description: "The eased output value, which is the result of applying the selected easing function to the input value.",
-      isContextful: true,
-      default: "0.0"
-    }
-  ],
-  description: "Applies an easing function to a value, transforming it based on the selected easing type. This is useful for creating smooth transitions in animations."
-}}/></a>
+:::note  
+Other nodes in the Animation category are also useful and can be combined to create unique effects.  
+:::
 
 <br/>
 
-Time node is essential to get the active frame or normalized time of the animation, while easing provides tons of built-in smoothing operations.
-Don't worry, we'll get to that soon.
+## Create a basic procedural animation
+
+Follow these steps to create a simple opacity animation:
 
-Basically all nodes in `Animation` category are your friends.
+1. Connect the **Normalized Time** node to **Output** under the image node. 
 
-## The simplest procedural animation
+  <Image src={simpleGraphImg} alt="Simple graph that connects Normalized Time of Time Node to Layer's Opacity input"/>
 
-<Image src={simpleGraphImg} alt="Simple graph that connects Normalized Time of Time Node to Layer's Opacity input"/>
+2. Use the **Normalized Time** output to track animation progress. For example, in a 2-second animation, the normalized time at 1 second is **0.5**. 
 
-Will produce 
+<br/>
+
+### The result
+This produces a smooth fade-in effect over the duration of the animation.
+  - At the beginning (0), the image is fully transparent.
+  - At the end (1), the image is fully opaque.
 
-<video src={resVideo} muted autoplay loop/>
+  <video src={resVideo} muted autoplay loop/>
 
 <br/>
 
-Neat, isn't it?
 
-Let's break down what happens here.
+## Creating a more complex procedural animation
```

Comment:
Titles are inconsistent in wording and capitalization

<img width="267" height="257" alt="Image" src="https://github.com/user-attachments/assets/a4b9f76f-5874-4dbb-b162-871826de9ecc" />